concourse/pipelines/create-deployer.yml

```diff
@@ -266,13 +267,17 @@ jobs:
             tar xzvf concourse-cert/concourse-cert.tar.gz
             if [ -f concourse.crt ] && [ -f concourse.key ]; then
               echo Certificate and private key already created, nothing to do
-              cp concourse-cert/concourse-cert.tar.gz .
+              cp concourse-cert/concourse-cert.tar.gz generated-concourse-cert
```

Comment:
Is it worth adding a trailing slash after the generated-concourse-cert directory name for clarity? I took me  a few seconds to realise that you weren't copying to a different file name.